third_party/move/move-bytecode-verifier/src/reference_safety/mod.rs

```diff
@@ -148,6 +148,7 @@ fn invoke_function(
     result_tys: Vec<SignatureToken>,
     meter: &mut impl Meter,
 ) -> PartialVMResult<()> {
+    let _func = verifier.stack.pop().unwrap();
```

Comment:
I see that code around here uses `safe_unwrap!`, suggesting that we might want to do the same here instead of the raw unwrap.